ignite/engine/engine.py

```diff
@@ -182,14 +190,14 @@ def __init__(self, **kwargs):
             if not hasattr(self, value):
                 setattr(self, value, 0)
 
-    def get_event_attrib_value(self, event_name):
+    def get_event_attrib_value(self, event_name: str) -> int:
```

Comment:
@justusschock I think here and after `event_name` can be `EventWithFilter`, `CallableEvents` or `Enum`

Reply:
Done